content/updates/2017-05-22-release-notes.md

```diff
@@ -0,0 +1,48 @@
++++
+date = "2017-05-22"
+title = "Platform Release Notes: May 22, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Researching
```

Comment:
Only because I was faster on the draw... Make it say what you think it should say!